apps/auction/src/hooks/api.ts

```diff
@@ -0,0 +1,33 @@
+import { useReadState, useSendMessage } from '@gear-js/react-hooks';
+import { useMemo } from 'react';
+import auctionMetaWasm from 'assets/wasm/dutch_auction.meta.wasm';
+import nftMetaWasm from 'assets/wasm/nft.meta.wasm';
+import { ADDRESS } from 'consts';
+import { AuctionState, NFTState } from 'types';
+import { Hex } from '@gear-js/api';
+
+function useAuction() {
+  const payload = useMemo(() => ({ Info: null }), []);
```

Comment:
maybe move to constants

Reply:
I prefer to not separate small encapsulated values